release.yaml

```diff
@@ -19,6 +19,8 @@ rancher-aks-operator-crd:
 rancher-csp-adapter:
 - 1.0.0+up1.0.0-rc2
 rancher-cis-benchmark:
-- 2.0.5-rc2
+- 2.0.5-rc3
 rancher-cis-benchmark-crd:
-- 2.0.5-rc2
+- 2.0.5-rc3
+system-upgrade-controller:
```

Comment:
Why are we adding system-upgrade-controller in this PR?

Reply:
Hi @prachidamle. I have not added it manually - it came as part of rebase from [upstream dev-v2.6](https://github.com/rancher/charts/blob/dev-v2.6/release.yaml#:~:text=system%2Dupgrade%2Dcontroller,%2D%20100.0.3%2Bup0.3.2) when I was trying to solve conflicts.